### PR TITLE
[control-plane-manager] Add etcd backup

### PR DESCRIPTION
**Pull request by alexey-gavrilov-flant**
## Description
Added cronjob that does etcd backup.

## Why do we need it, and what problem does it solve?
Cluster owners forget to configure backups in advance and in case of deleting objects from the cluster there is no way to restore them.

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [x] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: control-plane-manager
type: feature
summary: Add CronJob that does etcd backup.
impact_level: default
```

### CODE REVIEW

**Review by name212 — APPROVED** [review]

File: modules/040-control-plane-manager/docs/FAQ.md [hunk 1]
```diff
@@ -521,6 +521,10 @@ When deciding on the appropriate threshold values, consider resources consumed b
 
 ## etc backup and restore
 
+### What is done automatically
+
+CronJob `kube-system/d8-etcd-backup` is automatically started at 00:00 UTC+0. The result is saved in `/var/lib/etcd/etcd-backup.snapshot` on a random master in the cluster.
```
Comment (name212) [inline modules/040-control-plane-manager/docs/FAQ.md:526]:
```suggestion
CronJob `kube-system/d8-etcd-backup` is automatically started at 00:00 UTC+0. The result is saved in `/var/lib/etcd/etcd-backup.snapshot` on a random control-plane node in the cluster.
```

File: modules/040-control-plane-manager/docs/FAQ_RU.md [hunk 2]
```diff
@@ -514,6 +514,10 @@ spec:
 
 ## Резервное копирование etcd и восстановление
 
+### Что делается автоматически
+
+Автоматически запускается CronJob `kube-system/d8-etcd-backup` в 00:00 по UTC+0. Результат сохраняется в `/var/lib/etcd/etcd-backup.snapshot` на случайном master в кластере.
```
Comment (name212) [inline modules/040-control-plane-manager/docs/FAQ_RU.md:519]:
```suggestion
Автоматически запускается CronJob `kube-system/d8-etcd-backup` в 00:00 по UTC+0. Результат сохраняется в `/var/lib/etcd/etcd-backup.snapshot` на случайном мастер-узле.
```

**Review by z9r5 — APPROVED** [review]

File: modules/040-control-plane-manager/docs/FAQ_RU.md [hunk 3]
```diff
@@ -514,6 +514,10 @@ spec:
 
 ## Резервное копирование etcd и восстановление
 
+### Что делается автоматически
+
+Автоматически запускается CronJob `kube-system/d8-etcd-backup` в 00:00 по UTC+0. Результат сохраняется в `/var/lib/etcd/etcd-backup.snapshot` на случайном master-узле.
```
Comment (z9r5) [inline modules/040-control-plane-manager/docs/FAQ_RU.md:519]:
```suggestion
Автоматически запускается CronJob `kube-system/d8-etcd-backup` в 00:00 по UTC+0. Результат сохраняется в директории `/var/lib/etcd/etcd-backup.snapshot` на случайном master-узле.
```

File: modules/040-control-plane-manager/docs/FAQ.md [hunk 4]
```diff
@@ -521,6 +521,10 @@ When deciding on the appropriate threshold values, consider resources consumed b
 
 ## etc backup and restore
 
+### What is done automatically
+
+CronJob `kube-system/d8-etcd-backup` starts automatically at 00:00 UTC+0. The result is saved in `/var/lib/etcd/etcd-backup.snapshot` on a random control-plane node in the cluster.
```
Comment (z9r5) [inline modules/040-control-plane-manager/docs/FAQ.md:526]:
```suggestion
CronJob `kube-system/d8-etcd-backup` starts automatically at 00:00 UTC+0. The result is saved in `/var/lib/etcd/etcd-backup.snapshot` directory on a random control-plane node in the cluster.
```

**Review by z9r5 — APPROVED** [review]

File: modules/040-control-plane-manager/docs/FAQ.md [hunk 5]
```diff
@@ -521,6 +521,10 @@ When deciding on the appropriate threshold values, consider resources consumed b
 
 ## etc backup and restore
 
+### What is done automatically
+
+CronJob `kube-system/d8-etcd-backup` starts automatically at 00:00 UTC+0. The result is saved in `/var/lib/etcd/etcd-backup.snapshot` directory on a random control-plane node in the cluster.
```
Comment (RomanenkoDenys) [inline modules/040-control-plane-manager/docs/FAQ.md:526]: Not random. Result is saved on the master node where job performed.

**Review by RomanenkoDenys — CHANGES_REQUESTED** [review]

File: modules/040-control-plane-manager/docs/FAQ.md (hunk 5 above)
Comment (alexey-gavrilov-flant) [reply modules/040-control-plane-manager/docs/FAQ.md:526]: The job runs on a random master. More precisely on a less loaded master.

File: modules/040-control-plane-manager/docs/FAQ_RU.md [hunk 6]
```diff
@@ -514,6 +514,10 @@ spec:
 
 ## Резервное копирование etcd и восстановление
 
+### Что делается автоматически
+
+Автоматически запускаются CronJob `kube-system/d8-etcd-backup-*` в 00:00 по UTC+0. Результат сохраняется в `/var/lib/etcd/etcd-backup.snapshot` на всех нодах с `control-plane` в кластере.
```
Comment (name212) [inline modules/040-control-plane-manager/docs/FAQ_RU.md:519]:
```suggestion
Автоматически запускаются CronJob `kube-system/d8-etcd-backup-*` в 00:00 по UTC+0. Результат сохраняется в `/var/lib/etcd/etcd-backup.snapshot` на всех узлах с `control-plane` в кластере.
```

File: modules/040-control-plane-manager/docs/FAQ_RU.md [hunk 7]
```diff
@@ -514,6 +514,10 @@ spec:
 
 ## Резервное копирование etcd и восстановление
 
+### Что делается автоматически
+
+Автоматически запускаются CronJob `kube-system/d8-etcd-backup-*` в 00:00 по UTC+0. Результат сохраняется в `/var/lib/etcd/etcd-backup.snapshot` на всех узлах с `control-plane` в кластере.
```
Comment (z9r5) [inline modules/040-control-plane-manager/docs/FAQ_RU.md:519]:
```suggestion
Автоматически запускаются CronJob `kube-system/d8-etcd-backup-*` в 00:00 по UTC+0. Результат сохраняется в `/var/lib/etcd/etcd-backup.snapshot` на всех узлах с `control-plane` в кластере (мастер-узлы).
```

File: modules/040-control-plane-manager/docs/FAQ.md [hunk 8]
```diff
@@ -521,6 +521,10 @@ When deciding on the appropriate threshold values, consider resources consumed b
 
 ## etc backup and restore
 
+### What is done automatically
+
+CronJob `kube-system/d8-etcd-backup-*` is automatically started at 00:00 UTC+0. The result is saved in `/var/lib/etcd/etcd-backup.snapshot` on all nodes with `control-plane` in the cluster.
```
Comment (z9r5) [inline modules/040-control-plane-manager/docs/FAQ.md:526]:
```suggestion
CronJob `kube-system/d8-etcd-backup-*` is automatically started at 00:00 UTC+0. The result is saved in `/var/lib/etcd/etcd-backup.snapshot` on all nodes with `control-plane` in the cluster (master nodes).
```

**Review by z9r5 — APPROVED** [review]

**Pull request merged**